buildFonts.sh

```diff
@@ -119,7 +74,7 @@ if [[ $(docker images "$IMAGE" | wc -l) -lt 2 ]]; then
 fi
 
 CMDS="set -ex
-test -f MathJax-dev.tar.gz || wget -O MathJax-dev.tar.gz '${URL}'
+test -f MathJax-dev.tar.gz
 mk=\$(tar tf MathJax-dev.tar.gz | grep 'fonts/OTF/TeX/Makefile\$')
 tar xf MathJax-dev.tar.gz
```

Comment:
Shouldn't lines 77-79 (in new file) be removed?

Reply:
I'll give that a try.  I was trying to minimize the number of changes but if we can get rid of one of the tar/untar steps that would be good.